storage/src/cache/state/mod.rs

```diff
@@ -58,7 +58,7 @@ pub trait ChunkMap: Any + Send + Sync {
     /// Check whether the chunk is ready for use, and mark it as pending if not ready yet.
     ///
     /// The function returns:
-    /// - Err if timeout
+    /// - Ok(false) Waiting for inflight backend IO timeouts.
```

Comment:
Does the caller need to distinguish the case and 
`Ok(false) and also marks the chunk as pending, either set_ready_and_clear_pending() or clear_pending() must be called to clear the pending state`?

Reply:
Yes. Let's make it retry if waiting for inflight IO timeouts.